addons/survey/views/survey_templates.xml

```diff
@@ -336,6 +342,14 @@
             <t t-if="question.question_type == 'multiple_choice'"><t t-call="survey.question_multiple_choice"/></t>
             <t t-if="question.question_type == 'matrix'"><t t-call="survey.question_matrix"/></t>
             <div class="o_survey_question_error overflow-hidden border-0 py-0 px-3 alert alert-danger" role="alert"></div>
+            <div t-if="is_skipped_question" role="alert"
+                 class="d-flex align-items-center justify-content-between overflow-hidden border-0 p-3 alert alert-danger">
+                <span t-out="question.constr_error_msg or default_constr_error_msg"/>
+                <button t-if="survey.questions_layout == 'page_per_question'" class="btn btn-sm btn-link"
+                        type="submit" t-att-value="'finish' if len(skipped_questions) == 1 else 'next_skipped'">
```

Comment:
Also should we update the continue button at the bottom of the page too, like we do for sections? And/or maybe add text to specify what the arrow will do.